tests/unit/model-test.js

```diff
@@ -713,6 +713,52 @@ module('unit/model', function(hooks) {
     assert.equal(get(model, 'hb'), true, 'alias to missing with default');
   });
 
+  test('.unknownProperty resolves id-matched values to external m3-models when special ref present in _data passed to schema', function(
```

Comment:
I would shorten the test name to `.unknownProperty passes the model data hash to schema when resolving values`.

Reply:
Updated the test name.